sq: would mess if set key mapping to '' (#675)
preservim/tagbar: doc/tagbar.txt

These mappings can be redefined with the given map options. The argument can
be either a string or a |List| of strings. In the latter case the
functionality will be assigned to all of the keys in the list.
For example, if you want to remap the sort toggling functionality to "r":
>
        let g:tagbar_map_togglesort = "r"
<

> For example, or if you want to disable this mapping, then set to '':

Reviewer: ```suggestion
Alternatively, if you want to disable this mapping, then set to '':
```
